celer/tests/test_enet.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import numpy as np
+from numpy.testing import assert_allclose
+from sklearn.linear_model import enet_path, ElasticNet as sklearn_ElasticNet
+
+from celer import Lasso, ElasticNet, celer_path
+from celer.utils.testing import build_dataset
+
+
+def test_raise_errors_l1_ratio():
+    with np.testing.assert_raises(ValueError):
+        ElasticNet(l1_ratio=5.)
+
+    with np.testing.assert_raises(NotImplementedError):
+        X, y = build_dataset(n_samples=30, n_features=50)
+        y = np.sign(y)
+        celer_path(X, y, 'logreg', l1_ratio=0.5)
+
+
+@pytest.mark.parametrize("sparse_X", [True, False])
+def test_enet_lasso_equivalence(sparse_X):
+    n_samples, n_features = 30, 50
+    X, y = build_dataset(n_samples, n_features, sparse_X=sparse_X)
+
+    reg_1 = Lasso()
```

Comment:
it could be that for this dataset, alpha_max is lower than 1, and so here both model return 0 as `coef_`, so the test is not really doing what we aim for.
It's heavier but I do not see a better solution than to explicitely compute alpha_max for this data and set alpha as a fraction of it.